src/test/util/failpoint.rs

```diff
@@ -48,10 +48,20 @@ impl FailPoint {
         client: &Client,
         criteria: impl Into<Option<SelectionCriteria>>,
     ) -> Result<FailPointGuard> {
+        // TODO: DRIVERS-1385 remove this logic for moving errorLabels to the top level.
```

Comment:
This is necessary because the new spec test (`mongos-recovery-token`) sets up a failpoint with a `RetryableWriteError`, which requires `errorLabels` to be on the failpoint top level when it's initialized. This can also be observed [here](https://github.com/mongodb/mongo-rust-driver/blob/ab01ac04a08b40147a7fdbd5563c7d7cbcd7b159/src/test/spec/retryable_writes/mod.rs#L80)